### PR TITLE
Auth Context

### LINKED ISSUE #12: Implement Authentication
- Add UI for sign in and singup pages
- Create Auth context and use it across the app
- Use mockbee mock backend

### PR DESCRIPTION
**Pull request by ttahm3d**
- Handling authentication in context
- Both login and signup work
- Style changes to active links in header

### CODE REVIEW

File: src/pages/auth/Signup/Signup.jsx [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
     firstName: "",
     lastName: "",
     email: "",
     password: "",
     contact: "",
   });
+  const { authState, authDispatch } = useAuth();
+  const navigate = useNavigate();
+  // const [_, setUserToken] = useLocalStorage("user-token")
+
```
Comment (raman1999) [inline src/pages/auth/Signup/Signup.jsx:16]: remove this comments

File: src/pages/auth/Signup/Signup.jsx [hunk 2]
```diff
@@ -1,29 +1,62 @@
...
+    setSignupForm({
+      firstName: "",
+      lastName: "",
+      email: "",
+      password: "",
+      contact: "",
+    });
+  };
+
+  console.log(authState);
```
Comment (raman1999) [inline src/pages/auth/Signup/Signup.jsx]: remove console

**Review by raman1999 — APPROVED** [review]
I liked the way u break down your components, good naming conventions done, clean code written and good logical thinking done in creating functionalities.

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good.

**Pull request merged**